foxpuppet/windows/__init__.py

```diff
@@ -3,5 +3,5 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from foxpuppet.windows.base import BaseWindow  # noqa: F401
-from foxpuppet.windows.browser import BrowserWindow  # noqa: F401
+from foxpuppet.windows.browser.window import BrowserWindow  # noqa: F401
```

Comment:
Important classes under `foxpuppet.windows.browser.*` we should put into the __init__.py file for an easier import. For the window classes themselve I could even imagine to add them all to `foxpuppet/windows/__init__.py`, so we could do `from foxpuppet.windows import BrowserWindow`.